core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -73,34 +75,36 @@ private static String prependTitleIfRequired(String icon, String title) {
         return icon;
     }
 
-    public static String getIonicon(String name, String title) {
-        if (IONICONS.containsKey(name)) {
-            String icon = IONICONS.get(name);
-            return prependTitleIfRequired(icon, title);
+    // for Jelly
+    @Restricted(NoExternalUse.class)
+    public static String getSymbol(String name, String title) {
```

Comment:
Looking at the code I'm pretty sure this won't work for plugins wanting to add their own symbol.

Probably okay for this PR but should be addressed in a follow up